fleche/memo.ml

```diff
@@ -144,7 +144,7 @@ let interp_command ~st ~fb_queue stm : _ Stats.t =
       fb_queue := [];
       Stats.make ~time res
     | Coq.Protect.R.Completed _ as res ->
-      let () = HC.add !cache (stm, st) (stm_loc, res) in
+      HC.add !cache (stm, st) (stm_loc, res);
```

Comment:
That's using the let for a reason (robusnest of syntax)

Let's not spend time doing these pointless changes.